### PR TITLE
MPS-20: Adding Serializable support to v2 models

### PR DESCRIPTION
**Pull request by WhosNickDoglio**
#### Issue
[MPS-20](https://vimean.atlassian.net/browse/MPS-20)

#### Summary
Updated all the v2 models to implement `Serializable`. 

#### How to Test
I wrote some test files in the main app to make a network request and pass along the model to a new activity where I used `intent.getSerializableExtra` to confirm the model serialized correctly.  

Let me know if anyone wants me to supply the test files. 👍

### CODE REVIEW

File: models/src/main/java/com/vimeo/networking2/SeasonConnections.kt [hunk 1]
```diff
@@ -15,4 +16,9 @@ data class SeasonConnections(
     @Json(name = "videos")
     val videos: Connection? = null
 
-)
+): Serializable {
```
Comment (anthonycr) [inline models/src/main/java/com/vimeo/networking2/SeasonConnections.kt]: Formatting issue here, also noticed a few other DTOs with formatting issues, can you run the formatter on each DTO to be safe?

**Review by anthonycr — CHANGES_REQUESTED** [review]
Great work, can you run the formatter on the DTOs though? There's a number with formatting issues.

**Review by howardrigberg — COMMENTED** [review]
@WhosNickDoglio What did you use to generate the UIDs?

File: models/src/main/java/com/vimeo/networking2/SeasonConnections.kt (hunk 1 above)
Comment (WhosNickDoglio) [reply models/src/main/java/com/vimeo/networking2/SeasonConnections.kt]: Formatting updated 👍

**Comment by WhosNickDoglio** [comment]
> @WhosNickDoglio What did you use to generate the UIDs?

@howardrigberg 

I found this [plugin](https://plugins.jetbrains.com/plugin/13497-generateserialversionuid) that auto generates UIDs with a keyboard shortcut.

**Review by anthonycr — APPROVED** [review]

**Review by howardrigberg — APPROVED** [review]

**Pull request merged**